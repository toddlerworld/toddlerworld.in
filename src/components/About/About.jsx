import React from 'react';
import './About.scss';
import {Button} from '../Button/Button';
import {Link} from 'react-router-dom';

function About () {
    return (
        <div className='about-container' data-testid='about-container'>
            <h1 className='heading'>About us</h1>
            <hr className='hr-about-us' />
            <div className='about-section-1'>
                <img src='/images/about/first.webp' alt='Teacher showing how to write to a child'/>
                <div className='about-text'>
                    <h1>Our Team</h1>
                    <div className='about-button'>
                        <Link to="/our-team">
                            <Button
                                buttonStyle='primary-button'
                                buttonSize='large'
                            >
                                Learn More
                            </Button>
                        </Link>
                    </div>
                </div>
            </div>
            <div className='about-section-2'>
                <img src='/images/about/second.webp' alt='Children sitting on chairs and reading books' />
                <div className='about-text-left'>
                    <h1>Our Environment</h1>
                    <div className='about-button'>
                        <Link to="/our-environment">
                            <Button
                                buttonStyle='primary-button'
                                buttonSize='large'
                            >
                                Learn More
                            </Button>
                        </Link>
                    </div>
                </div>
            </div>
            <div className='about-section-3'>
                <img src='/images/about/third.webp' alt='Nine Children playing together'/>
                <div className='about-text'>
                    <h1>Our Activities</h1>
                    <div className='about-button'>
                        <Link to="/our-activities">
                            <Button
                                buttonStyle='primary-button'
                                buttonSize='large'
                            >
                                Learn More
                            </Button>
                        </Link>
                    </div>
                </div>
            </div>
        </div>
    );
}

export default About;