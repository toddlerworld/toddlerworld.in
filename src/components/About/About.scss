.about-container {
    padding: 4rem;
    position: relative;
}
.heading {
    padding-top: 2%;
    padding-left: 10%;
    justify-content: center;
    text-align: left;
}

.hr-about-us{
    border: 1px solid #e6e6e6;
    margin: 2rem 0;
    width:100%;
}

.about-section-2 {
    display: flex;
    flex-direction: row-reverse;
}

.about-section-1, .about-section-3 {
    padding-left: 10%;
    display: flex;
    flex-direction: row;
}

.about-section-1 h1 {
    justify-content: center;
    text-align: center;
}

.about-button {
    margin: 6px;
}

@media screen and (max-width: 768px) {
    .about-container {
        padding: 1rem;
    }
    .about-section-1, .about-section-2, .about-section-3 {
        display: flex;
        flex-direction: column;
        padding: 10%;
    }

    .about-section-1 h1, .about-section-2 h1, .about-section-3 h1 {
        padding-top: 5%;
    }
    .about-button {
        padding-top: 3%;
        padding-left: 33%;
        padding-right: 33%;
    }
}