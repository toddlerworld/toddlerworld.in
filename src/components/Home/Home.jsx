import React, {useEffect} from 'react';
import './Home.scss';
import Cards from './Card/Cards';
import MainSection from './MainSection/MainSection';
import Welcome from './Welcome/Welcome';
import HomeLandingImage from './Home-landing-image/Home-landing-image';
import Youtube from './Youtube/Youtube';
import Gallery from './Gallery/Gallery';
import Contact from './Contact/Contact';
import Reviews from './Reviews/Reviews';

const Home = () => {
    useEffect(() => {
        window.scrollTo(0, 0);
    }, []);
    return (
        <div className='home-page'>
            <img src="/images/main-section/landing-image.png" className="landing-photo" alt="Background Image" data-testid="landing-photo"/>
            <MainSection/>
            <Welcome/>
            <HomeLandingImage/>
            <Cards/>
            <Reviews/>
            <Youtube/>
            <Gallery/>
            <Contact/>
        </div>
    );
};
export default Home;