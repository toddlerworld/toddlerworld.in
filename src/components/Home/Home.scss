.landing-photo {
    object-fit: cover;
    width: 100%;
    height: 92%;
    position: fixed;
    z-index: -2;
}

//@media (max-width: 600px) {
//    .landing-photo {
//        content: url('../../../public/images/mobile.webp');
//    }
//}