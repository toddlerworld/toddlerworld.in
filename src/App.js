import React from 'react';
import './App.css';
import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
import Navbar from './components/Navbar/Navbar';
import Home from "./components/Home/Home";
import Header from "./components/Header/Header";
import Footer from "./components/Footer/Footer";

function App() {
    return (
        <>
            <Router>
                <Header />
                <Navbar />
                <Routes>
                    <Route path="/" element={<Home />} />
                </Routes>
                <Footer />
            </Router>
        </>
    );
}

export default App;
