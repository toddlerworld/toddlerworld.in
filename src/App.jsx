import React from 'react';
import './App.css';
import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
import Navbar from './components/Navbar/Navbar';
import Home from './components/Home/Home';
import Header from './components/Header/Header';
import Footer from './components/Footer/Footer';
import About from './components/About/About';
import Team from './components/Team/Team';
import Environment from './components/Environment/Environment';
import Welcome from './components/Welcome/Welcome';
import PlayEquipment from './components/PlayEquipment/PlayEquipment';
import ParentPartnership from './components/ParentPartnership/ParentPartnership';
import PoliciesProcedures from './components/PoliciesProcedures/PoliciesProcedures';
import Admissions from './components/Admissions/Admissions';
import Activities from './components/Activities/Activities';
import Careers from './components/Careers/Careers';
import PlayGroup from './components/PlayGroup/PlayGroup';
import PreNursery from './components/PreNursery/PreNursery';
import Nursery from './components/Nursery/Nursery';
import Kindergarten1 from './components/Kindergarten1/Kindergarten1';
import Kindergarten2 from './components/Kindergarten2/Kindergarten2';
import ContactUs from './components/Contact-us/Contact-us';
import Programmes from './components/Programmes/Programmes';
import NotFound from './components/NotFound/NotFound';

function App() {
    return (
        <>
            <Router>
                <Header />
                <Navbar />
                <Routes>
                    <Route path="/" element={<Home />} />
                    <Route path="/about-us" element={<About />} />
                    <Route path="/our-team" element={<Team />} />
                    <Route path="/our-environment" element={<Environment />} />
                    <Route path="/welcome" element={<Welcome />} />
                    <Route path="/play-equipment" element={<PlayEquipment />} />
                    <Route path="/parent-partnership" element={<ParentPartnership />} />
                    <Route path="/policies-procedures" element={<PoliciesProcedures />} />
                    <Route path="/admissions" element={<Admissions />} />
                    <Route path="/activities" element={<Activities />} />
                    <Route path="/careers" element={<Careers />} />
                    <Route path="/play-group" element={<PlayGroup />} />
                    <Route path="/pre-nursery" element={<PreNursery />} />
                    <Route path="/nursery" element={<Nursery />} />
                    <Route path="/kindergarten-1" element={<Kindergarten1 />} />
                    <Route path="/kindergarten-2" element={<Kindergarten2 />} />
                    <Route path="/contact-us" element={<ContactUs />} />
                    <Route path="/programmes" element={<Programmes />} />
                    <Route path="*" element={<NotFound />} />
                </Routes>
                <Footer />
            </Router>
        </>
    );
}

export default App;
