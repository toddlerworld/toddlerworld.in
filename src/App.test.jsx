import {render, screen} from '@testing-library/react'
import React from 'react'
import '@testing-library/jest-dom'
import App from './App'
import renderer from 'react-test-renderer';
import {MemoryRouter, Route, Routes} from 'react-router-dom';
import About from './components/About/About';
import Home from './components/Home/Home';

describe('App test', () => {
    // Test 1: Render Test
    test('renders without crashing', () => {
        render(<App />);
    });

    // Test 2: Check if the title is correct
    test('check if the title is correct', () => {
        render(<App />);
        expect(screen.getByText('Welcome to Toddler World')).toBeInTheDocument();
    });

    test('renders Home component for / path', () => {
        render(
            <MemoryRouter initialEntries={['/']}>
                <Routes>
                    <Route path="/" element={<Home />} />
                </Routes>
            </MemoryRouter>
        );

        expect(screen.getByText('Welcome to Toddler World')).toBeInTheDocument();
    });

    test('renders About component for /about-us path', () => {
        render(
            <MemoryRouter initialEntries={['/about-us']}>
                <Routes>
                    <Route path="/about-us" element={<About />} />
                </Routes>
            </MemoryRouter>
        );

        expect(screen.getByText('About us')).toBeInTheDocument();
    });
});