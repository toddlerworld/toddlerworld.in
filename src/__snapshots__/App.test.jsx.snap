// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test matches snapshot 1`] = `
Array [
  <div
    className="header-container"
  >
    <p>
      Book a visit by calling us on 
      <a
        className="telephone-number"
        href="tel:+917889744619"
      >
        7889744619
      </a>
      ,
      <a
        className="telephone-number"
        href="tel:+917051427879"
      >
         7051427879
      </a>
    </p>
  </div>,
  <nav
    className="navbar"
    data-testid="navbar-component"
  >
    <div
      className="navbar-container"
    >
      <a
        className="navbar-logo"
        href="/"
        onClick={[Function]}
      >
        <img
          alt="Logo"
          data-testid="menu-icon-component"
          src="/images/logo.png"
        />
      </a>
      <div
        className="menu-icon"
        data-testid="menu-icon"
        onClick={[Function]}
      >
        <i
          className="fas fa-bars"
        />
      </div>
      <ul
        className="nav-menu"
        data-testid="nav-menu-component"
      >
        <li
          className="nav-item"
        >
          <a
            className="nav-links"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-links"
            href="/about-us"
            onClick={[Function]}
          >
            About us
          </a>
        </li>
        <li
          className="dropdown"
        >
          <button
            className="nav-links"
          >
            Information
            <i
              className="fa fa-caret-down"
            />
          </button>
          <div
            className="dropdown-content"
          >
            <a
              href="#"
            >
              Link 1
            </a>
            <a
              href="#"
            >
              Link 2
            </a>
            <a
              href="#"
            >
              Link 3
            </a>
          </div>
        </li>
      </ul>
    </div>
  </nav>,
  <div
    className="hero-container"
    data-testid="main-section"
  >
    <video
      autoPlay={true}
      className="main-video"
      data-testid="main-video"
      loop={true}
      muted={true}
      src="/images/home-page-pic.mp4"
    />
    <h1>
      Welcome to Toddler World
    </h1>
    <div
      className="button-login"
    >
      <a
        href="/login"
        onClick={[Function]}
      >
        <button
          className="btn primary-button secondary-button"
        >
          Find out more!
        </button>
      </a>
    </div>
  </div>,
  <div
    className="welcome"
    data-testid="welcome-component"
  >
    <h2>
      Welcome To Toddler World
    </h2>
    <p
      className="welcome-paragraph"
    >
      Toddler World is a friendly, caring and committed facility in Srinagar, which provides developmental childcare, embracing excellence and quality. Our program is designed to foster the physical, cognitive, social, emotional and cultural development of each child in a loving and nurturing environment. We believe children learn through hands-on activities. Our nursery is specifically designed to allow free movement and exploration according to each childâ€™s abilities. Its large, bright, colourful playrooms and outdoor play areas accessorised by state-of-the-art equipment and age-appropriate toys create a magical haven for your child. 
    </p>
    <a
      href="/admissions"
      onClick={[Function]}
    >
      <button
        className="btn primary-button secondary-button"
      >
        Register your child!
      </button>
    </a>
  </div>,
  <div
    className="home-landing-container"
    data-testid="home-landing-image"
  >
    <h1>
      ENABLING A SUCCESSFUL TRANSITION TO A PRIMARY SCHOOL
    </h1>
  </div>,
  <div
    className="cards"
  >
    <h1>
      A bit about us
    </h1>
    <div
      className="card"
    >
      <div
        className="wrapper"
      >
        <ul
          className="items"
        >
          <li
            className="item"
          >
            <a
              className="link"
              href="/our-environment"
              onClick={[Function]}
            >
              <figure
                className="image"
                data-category="Our Environment"
              >
                <img
                  alt="Job Image"
                  className="img-item"
                  src="/gifs/placeholder1.gif"
                />
              </figure>
              <div
                className="aboutCard"
              >
                <h5
                  className="caption"
                >
                  Our well-designed environment supports child's growth and development
                </h5>
              </div>
            </a>
          </li>
          <li
            className="item"
          >
            <a
              className="link"
              href="/play-equipment"
              onClick={[Function]}
            >
              <figure
                className="image"
                data-category="Our Facilities"
              >
                <img
                  alt="Job Image"
                  className="img-item"
                  src="/gifs/placeholder2.gif"
                />
              </figure>
              <div
                className="aboutCard"
              >
                <h5
                  className="caption"
                >
                  World class infrastructure for holistic development of the child
                </h5>
              </div>
            </a>
          </li>
          <li
            className="item"
          >
            <a
              className="link"
              href="/login"
              onClick={[Function]}
            >
              <figure
                className="image"
                data-category="Our Programmes"
              >
                <img
                  alt="Job Image"
                  className="img-item"
                  src="/gifs/placeholder3.gif"
                />
              </figure>
              <div
                className="aboutCard"
              >
                <h5
                  className="caption"
                >
                  Our age appropriate programmes that grow with your child
                </h5>
              </div>
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>,
  <div
    className="youtube-container"
    data-testid="youtube-component"
  >
    <h1>
      YOU'LL LOVE WHY WE'RE DIFFERENT. TAKE A LOOK
    </h1>
    <hr
      className="hr-youtube"
    />
    <p>
      Toddler World is renowned for providing rich learning opportunities in unique and exciting environment. 
    </p>
    <div
      className="youtube-video-container"
    >
      <iframe
        className="youtube-video"
        data-testid="youtube-video"
        src="https://www.youtube.com/embed/FxxsA-qaHTM"
      />
    </div>
  </div>,
  <div
    className="gallery"
    data-testid="gallery-component"
  >
    <h1>
      Photo Gallery
    </h1>
    <hr
      className="hr-gallery"
    />
    <div
      className="carousel-root crsl"
    >
      <div
        className="carousel carousel-slider"
        style={
          Object {
            "width": "100%",
          }
        }
      >
        <button
          aria-label="previous slide / item"
          className="control-arrow control-prev"
          onClick={[Function]}
          type="button"
        />
        <div
          className="slider-wrapper axis-horizontal"
          style={Object {}}
        >
          <ul
            className="slider animated"
            onMouseDown={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "MozTransitionDuration": "350ms",
                "OTransform": "translate3d(-70%,0,0)",
                "OTransitionDuration": "350ms",
                "WebkitTransform": "translate3d(-70%,0,0)",
                "WebkitTransitionDuration": "350ms",
                "msTransform": "translate3d(-70%,0,0)",
                "msTransitionDuration": "350ms",
                "transform": "translate3d(-70%,0,0)",
                "transitionDuration": "350ms",
              }
            }
          >
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="Children dressed up as part of an activity"
                src="/images/home-page-gallery/18.webp"
              />
            </li>
            <li
              className="slide selected previous"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="A child cutting a loaf of bread with a toy knife in a playful kitchen setting"
                src="/images/home-page-gallery/1.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="A child peering with curiosity between the pages of the Rocky Monkey book."
                src="/images/home-page-gallery/2.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="Treading Beads at Toddler World"
                src="/images/home-page-gallery/3.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="A young child happily stacking colorful building blocks on a sunny day, illustrating the joy of creative play and early learning."
                src="/images/home-page-gallery/4.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="Two joyful boys engaged in imaginative play with toy cars."
                src="/images/home-page-gallery/5.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="A joyful boy with paint on his hands creating a colourful triangle with his hand and looking through it."
                src="/images/home-page-gallery/6.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="A child in costume gazing up at the sky."
                src="/images/home-page-gallery/7.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="A child in costume looking at the camera."
                src="/images/home-page-gallery/8.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="Child selling potatoes, peppers, and beans while holding money in hand."
                src="/images/home-page-gallery/9.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="Two boys using the Rock Climbing facilities"
                src="/images/home-page-gallery/10.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="Two children playing inside the play house"
                src="/images/home-page-gallery/11.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="A child riding a honey in the play area"
                src="/images/home-page-gallery/12.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="An excited child ready to take the blocks out of the bag"
                src="/images/home-page-gallery/13.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="Children enjoying playing in the pool area"
                src="/images/home-page-gallery/14.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="Children enjoying playing in the ball area"
                src="/images/home-page-gallery/15.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="A child reading about shapes from the book"
                src="/images/home-page-gallery/16.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="Children sitting on chairs and reading books"
                src="/images/home-page-gallery/17.webp"
              />
            </li>
            <li
              className="slide"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="Children dressed up as part of an activity"
                src="/images/home-page-gallery/18.webp"
              />
            </li>
            <li
              className="slide selected previous"
              onClick={[Function]}
              style={
                Object {
                  "minWidth": "80%",
                }
              }
            >
              <img
                alt="A child cutting a loaf of bread with a toy knife in a playful kitchen setting"
                src="/images/home-page-gallery/1.webp"
              />
            </li>
          </ul>
        </div>
        <button
          aria-label="next slide / item"
          className="control-arrow control-next"
          onClick={[Function]}
          type="button"
        />
      </div>
      <div
        className="carousel"
      >
        <div
          className="thumbs-wrapper axis-vertical"
        >
          <button
            aria-label="previous slide / item"
            className="control-arrow control-prev control-disabled"
            onClick={[Function]}
            type="button"
          />
          <ul
            className="thumbs animated"
            onMouseDown={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "MozTransform": "translate3d(0,0,0)",
                "MozTransitionDuration": "350ms",
                "MsTransform": "translate3d(0,0,0)",
                "MsTransitionDuration": "350ms",
                "OTransform": "translate3d(0,0,0)",
                "OTransitionDuration": "350ms",
                "WebkitTransform": "translate3d(0,0,0)",
                "WebkitTransitionDuration": "350ms",
                "msTransform": "translate3d(0,0,0)",
                "msTransitionDuration": "350ms",
                "transform": "translate3d(0,0,0)",
                "transitionDuration": "350ms",
              }
            }
          >
            <li
              aria-label="slide item 1"
              className="thumb selected"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="A child cutting a loaf of bread with a toy knife in a playful kitchen setting"
                src="/images/home-page-gallery/1.webp"
              />
            </li>
            <li
              aria-label="slide item 2"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="A child peering with curiosity between the pages of the Rocky Monkey book."
                src="/images/home-page-gallery/2.webp"
              />
            </li>
            <li
              aria-label="slide item 3"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="Treading Beads at Toddler World"
                src="/images/home-page-gallery/3.webp"
              />
            </li>
            <li
              aria-label="slide item 4"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="A young child happily stacking colorful building blocks on a sunny day, illustrating the joy of creative play and early learning."
                src="/images/home-page-gallery/4.webp"
              />
            </li>
            <li
              aria-label="slide item 5"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="Two joyful boys engaged in imaginative play with toy cars."
                src="/images/home-page-gallery/5.webp"
              />
            </li>
            <li
              aria-label="slide item 6"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="A joyful boy with paint on his hands creating a colourful triangle with his hand and looking through it."
                src="/images/home-page-gallery/6.webp"
              />
            </li>
            <li
              aria-label="slide item 7"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="A child in costume gazing up at the sky."
                src="/images/home-page-gallery/7.webp"
              />
            </li>
            <li
              aria-label="slide item 8"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="A child in costume looking at the camera."
                src="/images/home-page-gallery/8.webp"
              />
            </li>
            <li
              aria-label="slide item 9"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="Child selling potatoes, peppers, and beans while holding money in hand."
                src="/images/home-page-gallery/9.webp"
              />
            </li>
            <li
              aria-label="slide item 10"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="Two boys using the Rock Climbing facilities"
                src="/images/home-page-gallery/10.webp"
              />
            </li>
            <li
              aria-label="slide item 11"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="Two children playing inside the play house"
                src="/images/home-page-gallery/11.webp"
              />
            </li>
            <li
              aria-label="slide item 12"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="A child riding a honey in the play area"
                src="/images/home-page-gallery/12.webp"
              />
            </li>
            <li
              aria-label="slide item 13"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="An excited child ready to take the blocks out of the bag"
                src="/images/home-page-gallery/13.webp"
              />
            </li>
            <li
              aria-label="slide item 14"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="Children enjoying playing in the pool area"
                src="/images/home-page-gallery/14.webp"
              />
            </li>
            <li
              aria-label="slide item 15"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="Children enjoying playing in the ball area"
                src="/images/home-page-gallery/15.webp"
              />
            </li>
            <li
              aria-label="slide item 16"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="A child reading about shapes from the book"
                src="/images/home-page-gallery/16.webp"
              />
            </li>
            <li
              aria-label="slide item 17"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="Children sitting on chairs and reading books"
                src="/images/home-page-gallery/17.webp"
              />
            </li>
            <li
              aria-label="slide item 18"
              className="thumb"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              style={
                Object {
                  "width": 80,
                }
              }
              tabIndex={0}
            >
              <img
                alt="Children dressed up as part of an activity"
                src="/images/home-page-gallery/18.webp"
              />
            </li>
          </ul>
          <button
            aria-label="next slide / item"
            className="control-arrow control-next control-disabled"
            onClick={[Function]}
            type="button"
          />
        </div>
      </div>
    </div>
  </div>,
  <div
    className="container"
    data-testid="contact-container"
  >
    <div
      className="contact-container"
    >
      <h1>
        Contact Us
      </h1>
      <hr
        className="hr-contact"
      />
      <h2>
        Better yet, see us in person!
      </h2>
      <p
        className="contact-paragraph"
      >
        We would love to see you in person. So feel free to call us to book an appointment.
      </p>
      <h2>
         Toddler World
      </h2>
      <p
        className="contact-paragraph"
      >
        Airport Avenue, Near Railway Bridge, Srinagar, Jammu and Kashmir, 190021
        <br />
      </p>
      <p
        className="contact-telephone"
      >
        Phone: 
        <a
          className="phone"
          href="tel:+911942303601"
        >
          +91 1942-303601
        </a>
        , 
        <a
          className="phone"
          href="tel:+911942303611"
        >
          +91 1942-303611
        </a>
        <br />
        Mobile: 
        <a
          className="phone"
          href="tel:+917051427879"
        >
          +91 705-1427879
        </a>
        , 
        <a
          className="phone"
          href="tel:+917889744619"
        >
          +91 788-9744619
        </a>
      </p>
      <p
        onClick={[Function]}
        style={
          Object {
            "cursor": "pointer",
          }
        }
      >
        <span
          style={
            Object {
              "fontWeight": "bold",
            }
          }
        >
          Monday
          :
        </span>
         
        10:00 am â€“ 04:00 pm
         
        <span>
          â–¼
        </span>
      </p>
    </div>
    <div
      className="maps-container"
    >
      <iframe
        src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3306.730162425703!2d74.7713164!3d34.025136700000004!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x38e191dba7d84fa3%3A0x16d8642869049f5a!2sToddler%20World!5e0!3m2!1sen!2suk!4v1695935838910!5m2!1sen!2suk"
      />
    </div>
  </div>,
  <div
    className="footer-container"
  >
    <section
      className="logo-socials"
    >
      <div
        className="wrapper-socials"
      >
        <a
          className="social-logo"
          href="/"
          onClick={[Function]}
        >
          Toddler World
        </a>
        <small
          className="website-rights"
        >
          Toddler World Â© 2016-2023
        </small>
        <div
          className="logos-socials"
        >
          <a
            aria-label="Facebook"
            className="social-icon-link facebook"
            href="https://www.facebook.com/toddlerworldsgr/"
            target="_blank"
          >
            <i
              className="fab fa-facebook-f"
            />
          </a>
          <a
            aria-label="Instagram"
            class="social-icon-link instagram"
            href="https://www.instagram.com/toddlerworldsgr/"
            target="_blank"
          >
            <i
              className="fab fa-instagram"
            />
          </a>
          <a
            aria-label="Youtube"
            class="social-icon-link youtube"
            href="https://www.youtube.com/@ToddlerWorld"
            target="_blank"
          >
            <i
              className="fab fa-youtube"
            />
          </a>
          <a
            aria-label="Twitter"
            class="social-icon-link twitter"
            href="https://www.twitter.com/toddlerworldsgr/"
            target="_blank"
          >
            <i
              className="fab fa-twitter"
            />
          </a>
          <a
            aria-label="LinkedIn"
            class="social-icon-link twitter"
            href="https://www.linkedin.com/company/toddlerworld/"
            target="_blank"
          >
            <i
              className="fab fa-linkedin"
            />
          </a>
        </div>
      </div>
    </section>
  </div>,
]
`;
